### ISSUE TITLE
Instructions are Linux exclusive

### ISSUE THREAD

**Issue by Nat-Tebb** [grpc/grpc.io#968]
The instructions appear to be solely intended for users of the linux OS, and don't generally seem to work for Windows users, can a separate instruction set be created for Windows users?

**Comment by ejona86** [comment]
What language are you interested in?

**Comment by Energywelder** [comment]
> What language are you interested in?

English is preferred please

**Comment by Nat-Tebb** [comment]
> I was asking about which programming language.

I might be dense, but I was talking solely about operating systems? All i know is the command prompt lines in the instruction set werent working in my windows command prompt, so I was hoping for a translation from Linux instructions to windows instructions?

**Comment by ejona86** [comment]
Right, but we support many different programming languages and some languages are much more likely to want to run on Windows or need different commands on Windows. I'm basically asking, "what page were you looking at that you wished had Windows instructions."

**Comment by ejona86** [comment]
Okay. Glancing at just the quickstart, looks like adding .bat for multiple of the commands would probably be enough. Maybe some of them need / replaced with \ since Windows is finicky on when it will accept forward slashes.
